settings_prd.md

```
@@ -0,0 +1,160 @@
+    - caching_prompt: bool
+    - disable_vision: bool | None
+    - seed: int | None
+    - safety_settings: list[dict[str, str]] | None
+    - usage_id: string (defaults to 'default')
+    - metadata: dict[str, any]
+  - Serialization/secret handling
+    - api_key and select fields are masked by default; can be exposed with context={'expose_secrets': True}
+- Current extension behavior
+  - Hardcoded defaults in ConnectionManager when starting a conversation
+    - model: litellm_proxy/anthropic/claude-sonnet-4-20250514
+    - base_url: https://llm-proxy.eval.all-hands.dev
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)